src/views/Notifications/Settings/index.tsx

```diff
@@ -85,6 +96,13 @@ const NotificationSettings = () => {
       ...dirtyInputs
     };
 
+    // if datesChangedNotificationType is not changed by user, but datesChanged is changed, then do the following logic
+    if (!changes.datesChangedNotificationType && changes.datesChanged?.length) {
+      // If Dates Changed notification is subscribed, then manually set datesChangedNotificationType to ALL_MODELS
+      changes.datesChangedNotificationType =
```

Comment:
There's a few instances where consistency with capitalization in comments is off.  Not huge just a nitpick